leeJungWoo/app.js

```diff
@@ -6,15 +6,15 @@ const { DataSource } = require('typeorm');
 
 dotenv.config();
 
-const myDatasource = new DataSource({
+const mysqlDatasource = new DataSource({
```

Comment:
```suggestion
const appDatasource = new DataSource({
```

데이터베이스 클라이언트에 대한 정보도 노출하기 않기 위해 환경변수 파일에서 데이터베이스의 타입도 관리하고 있습니다.

`mysqlDatasource`라는 변수명 보다는 다른 변수명을 사용해주세요!